src/common/domain/errors/delegated-booking-decompression-error.ts

```diff
@@ -0,0 +1,6 @@
+export class DelegatedBookingDecompressionError extends Error {
+  constructor() {
+    super();
+    Object.setPrototypeOf(this, DelegatedBookingDecompressionError.prototype);
```

Comment:
What's the purpose of this?

Reply:
I emulated the same pattern as the handling in the Journal service.

See here:  https://github.com/dvsa/mes-journal-service/blob/develop/src/common/domain/errors/journal-decompression-error.ts